### PR TITLE
[stable/redis] Support setting the volume mount path

### PR DESCRIPTION
**Pull request by Marlinc**
I'm using the official Redis image which uses `/data` as path for the Redis storage.

### CODE REVIEW

**Comment by sameersbn** [comment]
I see that this change will allow you to mount the persistent volume at `/data` while using the official docker image. But I wonder if this parameter would give users the idea that location of the persistent volume can be changed in general. what do think about this @prydonius ?

**Comment by prydonius** [comment]
@sameersbn good point, perhaps a comment in the values.yaml could address the intended usecase for this, e.g. "use a different mount point for a different Redis image"

**Comment by sameersbn** [comment]
@Marlinc could you incorporate the suggestions made by @prydonius? more importantly please make a comment in the README.

**Comment by Marlinc** [comment]
I've added the parameter to the README file and have changed the comment in `values.yaml` a bit. I'm unsure about the test error as I have actually upped the version already.

**Comment by prydonius** [comment]
@Marlinc looks like another PR got redis to 1.1.6 before you, can you rebase and bump this to 1.1.7? thanks!

**Comment by Marlinc** [comment]
Ah okay! Will do that

File: stable/redis/values.yaml [hunk 1]
```diff
@@ -40,6 +40,10 @@ args:
 persistence:
   enabled: true
 
+  ## The path the volume will be mounted at, useful when using different
+  ## Redis packages.
```
Comment (prydonius) [inline stable/redis/values.yaml]: "redis packages" sounds a little confusing to me, I think "image" singular is more clear.

Comment (Marlinc) [reply stable/redis/values.yaml]: You're right, not sure where that came from

**Review by prydonius — APPROVED** [review]
/lgtm

**Pull request merged**